### ISSUE TITLE
inception3 jpg loading

### ISSUE THREAD

**Issue by andsynchrony** [memo/ofxMSATensorFlow#5]
I wasn't able to run the inception3 example at all, so I built the current version of TensorFlow and updated the header files. I was able to run the example as well as load jpg images after that. Still having random crashes though. Resizing all ofImages to the setting used in the classifier helped a little. You can check out this fork and see if you're able to load jpgs with it:
https://github.com/andsynchrony/ofxMSATensorFlow
Updated library file can be found under releases as well:
https://github.com/andsynchrony/ofxMSATensorFlow/releases

**Comment by alesaccoia** [comment]
+1
It must be something about the ctor in tensorflow::ops::Const... is the OSX lib v0.7 zip in sync with the include files in this repo?
I'll try @andsynchrony's branch + lib as well.
![stack trace](https://cloud.githubusercontent.com/assets/1385023/15050194/7940982a-12f2-11e6-9292-135802da5b98.png)

**Comment by alesaccoia** [comment]
I got a bit further down get_top_scores with @andsynchrony but still crashing, on     
       tensor_to_vector(output_tensors[0], out_scores);
with output_tensors having size 0.
Noticed that during loading I get 
[notice ] Initializing...
.
.
[notice ] 1003:softmax
[ error ] Error running network for weights and biases variable hack

**Comment by memo** [comment]
Hey, just a reminder that you'll get crashes if you run a debug build with the release lib. Have you tried a release build?

**Comment by andsynchrony** [comment]
Hey memo, thanks for your reply. With my repository, I tried Release as well as Debug configuration. I also tried to build the lib for Debug, however without success. Unfortunately, it also crashes in the Release config from time to time. But jpg images are working after compiling TensorFlow 0.8 release!

**Comment by memo** [comment]
hm ok. I haven't tested with 0.8 yet. Will do and have a look. Though I don't have access to OSX at the moment, only linux.

**Comment by alesaccoia** [comment]
I get the same exact problem also in Release Mode with 0.7. I still haven't been able to find a workaround

**Comment by memo** [comment]
Hey I'm updating the addon to use tensorflow r1.0, and there is an awful lot to fix. They've introduced a tensorflow::Scope and everything generally goes through that now. I have some other issues now but once that's done hopefully I'll be able to push soon.
